aussie_clock.py

```diff
@@ -114,6 +115,25 @@ async def goob_schedule_upd():
                         await channel.send(schedule_message)
 
 
+async def birthday_check():
+    birthdays = json.loads(open("birthdays.json", "r").read())
+    for b_day in birthdays:
+        user = client.get_user(b_day)
+        today = datetime.today()
+        b_day = datetime.fromtimestamp(birthdays[b_day])
+        if today.month == b_day.month:
+            if today.day == b_day.day:
+                if today.hour == 14 and today.minute == 0:
```

Comment:
Why `14` and `0`? These particular values aren't clear - perhaps add a comment explaining why they are used? My best guess is that you only want to say "happy birthday" to users once per day, but if so, why is this function called so often (guessing every minute)?

Reply:
Its called every minute because its being run along with 2 other functions (Aussie Clock and Goob Schedule) which also run every minute

Reply:
In that case, why check the same thing for every user whose birthday is today - you could just check the time at the start of the function

Reply:
ohh i see, move it outside the for loop, that way it saves processing power if it didn't need to do it in the first place

Reply:
i mean its already being checked with the actual date and time so i might as well keep it there